### PR TITLE
Make installable as a ZSH plugin by o-m-z compatible ZSH frameworks

### PR DESCRIPTION
**Pull request by unixorn**
Add ZSH plugin support. This lets git-aware frameworks like [Antigen](https://github.com/zsh-users/antigen) and [Zgen](https://github.com/tarjoilija/zgen) handle automatically checking for updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,30 @@ Type `git open` to open the GitHub page or website for a repository in your brow
...
+
+Add `antigen bundle paulirish/git-open` to your `.zshrc` with your other bundle commands.
+
+Antigen will handle cloning the plugin for you automatically the next time you start zsh, and periodically checking for updates to the git repository. You can also add the plugin to a running zsh with `antigen bundle paulirish/git-open` for testing before adding it to your `.zshrc`.
+
+### [Oh-My-Zsh](http://ohmyz.sh/)
+
+1. `cd ~/.oh-my-zsh/custom/plugins`
+2. `git clone git@github.com:paulirish/git-open.git gitopen`
+3. Add git-open to your plugin list - edit `~.zshrc` and change `plugins=(...)` to `plugins=(... gitopen)`
```
Comment (davidosomething) [inline README.md]: Please fix this .zshrc path, missing slash

File: git-open.plugin.zsh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# This plugin is MIT licensed to match the git-open license.
+#
+# Make git-open easy to install and keep up to date if you're using a
+# ZSH framework like Zgen or Antigen
+
+  export PATH=${PATH}:$(dirname $0)
```
Comment (davidosomething) [inline git-open.plugin.zsh]: I think you should move the dirname path before existing $PATH, like the omz npm plugin:

https://github.com/robbyrussell/oh-my-zsh/blob/286c3e5e28b651c3f629e6aaba589a1cf56da4d8/plugins/npm/npm.plugin.zsh
and the omz pyenv plugin:
https://github.com/robbyrussell/oh-my-zsh/blob/83cf8dc16f51babbb0193c5b97e568739c1f40de/plugins/pyenv/pyenv.plugin.zsh#L19

If people use this plugin via omz/antigen/etc. they probably expect it to be the priority "git open", where they may have a different git-open elsewhere in their $PATH

So just swap to `export PATH="$(dirname $0):${PATH}"`

File: README.md (hunk 1 above)
Comment (unixorn) [inline README.md]: Fixed, thanks.

**Comment by davidosomething** [comment]
Thanks for updating! lgtm

**Comment by faridvatani** [comment]
Before adding `plugins=(git-open)` to plugins, I already have `plugins=(git)` in my .zshrc , Should I delete git in my plugins? How to import it inside the git plugin?

**Pull request merged**